Doc/library/sqlite3.rst

```diff
@@ -966,18 +966,11 @@ Cursor Objects
 
    .. attribute:: rowcount
 
-      Although the :class:`Cursor` class of the :mod:`sqlite3` module implements this
-      attribute, the database engine's own support for the determination of "rows
-      affected"/"rows selected" is quirky.
-
-      For :meth:`executemany` statements, the number of modifications are summed up
-      into :attr:`rowcount`.
```

Comment:
We could keep this sentence.

Reply:
I'm not sure it adds anything to the existing sentence, and it would need to be rewritten, but I'm not the subject matter expert here...